### DISCUSSION TITLE
Sonic Velocity lab test

### DISCUSSION THREAD

**Discussion by m-sams-artc** [australiangeomechanics/agsformat#42] [Q&A]
Does the current ags format (with or without AU localisation) support test results from sonic velocity lab tests? This is a test with growing popularity for lab testing of sonic velocity. A non-destructive test which is done usually prior to a destructive UCS being done on the same sample - provides a good calibration between sonic velocity and UCS (paper - https://core.ac.uk/download/pdf/21747136.pdf). It is almost always paired with a UCS test, maybe it could go in RUCS group.

**Comment by SolidRockProgrammer** [comment]
Sonic is P-wave? P-wave is transmitted in groups
LDYN	Dynamic Testing
LDYT	Dynamic Testing - Test Data
